examples/boids/main.rs

```diff
@@ -369,15 +354,17 @@ fn main() {
                     let mut cpass = command_encoder.begin_compute_pass();
                     cpass.set_pipeline(&compute_pipeline);
                     cpass.set_bind_group(0, &particle_bind_groups[frame_num % 2], &[]);
-                    cpass.dispatch(NUM_PARTICLES, 1, 1);
+                    cpass.dispatch(INVOCATION_COUNT, 1, 1);
```

Comment:
this specifies the number of thread **groups**. So if each group is 64x1x1, then the group count will be NUM_PARTICLES / 64, rounded up.